gitmanager.py

```diff
@@ -156,11 +157,12 @@ def add_to_blacklist(cls, **kwargs):
 
                 payload = {"title": u"{0}: {1} {2}".format(username, op.title(), item),
                            "body": u"[{0}]({1}) requests the {2} of the {3} {4}. See the Metasmoke search [here]"
-                                   "(https://metasmoke.erwaysoftware.com/search?utf8=%E2%9C%93{5}{6})\n"
+                                   "(https://metasmoke.erwaysoftware.com/search?utf8=%E2%9C%93{5}{6}), and the"
```

Comment:
Can you 🔥 the `,` here? I think it would read better as `See the metasmoke search [here] and the Stack Exchange search [here]`.

Reply:
I agree; that reads a little better, and I think the comma was grammatically incorrect.  Thanks!